multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.121.0-core.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    logicalFilePath="org/cloudfoundry/multiapps/controller/core/db/changelog/db-changelog-1.110.0-core.xml"
```

Comment:
this line is not necessary (see https://docs.liquibase.com/concepts/advanced/logicalfilepath.html);
if you leave it, make sure to fix the version.